pennylane_cirq/simulator_device.py

```diff
@@ -132,9 +132,10 @@ def apply(self, operations, **kwargs):
         # pylint: disable=missing-function-docstring
         super().apply(operations, **kwargs)
 
-        # We apply an identity gate to all wires, otherwise Cirq would ignore
+        # We apply identity gates to all wires, otherwise Cirq would ignore
```

Comment:
```suggestion
        # TODO: remove the need for this hack by keeping better track of unused wires
        # We apply identity gates to all wires, otherwise Cirq would ignore
```